Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -199,7 +199,41 @@ final class CacheAdvanceTests: XCTestCase {
         XCTAssertEqual(Array(lorumIpsumMessages.dropFirst(lorumIpsumMessages.count - messages.count)), messages)
     }
 
+    func test_append_canWriteMessagesToCacheCreatedByADifferentCache() throws {
+        func createCache() throws -> CacheAdvance<TestableMessage> {
+            return try CacheAdvance<TestableMessage>(
+            file: testFileLocation,
+            maximumBytes: try requiredByteCount(for: lorumIpsumMessages, cacheWillOverwriteOldestMessages: true) / 3,
+            shouldOverwriteOldMessages: true)
+        }
+        let cache = try createCache()
+        for message in lorumIpsumMessages {
+            try cache.append(message: message)
+        }
+
+        let cachedMessages = try cache.messages()
+        let secondCache = try createCache()
+        try secondCache.append(message: "test")
+        XCTAssertEqual(cachedMessages + ["test"], try secondCache.messages())
+    }
+
     func test_messages_canReadMessagesWrittenByADifferentCache() throws {
```

Comment:
This method is technically a subset of the prior test... but good to call it out explicitly